src/test/scala/uk/gov/hmrc/ct/accounts/frs10x/AccountsFreeTextValidationFixture.scala

```diff
@@ -65,7 +65,7 @@ trait AccountsFreeTextValidationFixture extends WordSpec with Matchers with Mock
   def testAccountsCoHoTextFieldValidation(boxId: String, builder: (Option[String]) => ValidatableBox[AbridgedAccountsBoxRetriever]): Unit = {
     setUpMocks()
     "fail validation if invalid characters" in {
-      builder(Some("^ §")).validate(boxRetriever) shouldBe Set(CtValidation(Some(boxId), s"error.$boxId.regexFailure", Some(List("^, §"))))
+      builder(Some("^ §")).validate(boxRetriever) shouldBe Set(CtValidation(Some(boxId), s"error.$boxId.regexFailure", Some(List("^  §"))))
```

Comment:
yes that's part of the requirements. Kaye asked for a double space between illegal characters